docs/source/testing/graphql-playground.mdx

```diff
@@ -3,7 +3,11 @@ title: GraphQL Playground
 description: Visually explore Apollo Server
 ---
 
-import { Link } from 'gatsby';
+> ⚠️ **Apollo Server 2's GraphQL playground feature has an end-of-life date of 31 December 2022.**
+>
+>[Learn more about this deprecation and end-of-life.](/apollo-server/other-versions#apollo-server-2)
+
+ import { Link } from 'gatsby';
```

Comment:
This is showing up as text. Maybe needs to be first or maybe needs to be unindented or both